apis/nucleus/src/index.js

```diff
@@ -266,9 +267,9 @@ function nuked(configuration = {}) {
       },
       /**
        * Gets the app selections of this instance.
-       * @returns {Promise<AppSelections>}
+       * @returns {AppSelections}
        * @example
-       * const selections = await n.selections();
+       * const selections = n.selections();
        * selections.mount(element);
        */
       selections: async () => {
```

Comment:
this still return a promise.

Reply:
Yes it does, but looking at what it actually returns, I see no need for the function to be async actually...
It might break something though, to change the function into being synchronous.

Reply:
yea, that is what confused me. Does an async function always return a promise wrapping whatever you return?

Reply:
I can see now why it was a Promise, due to the:
// const appSelections = await root.getAppSelections(); // Don't expose this for now
 Should I make the Field one a promise as well, just be to consistent?

Reply:
> Does an async function always return a promise wrapping whatever you return?
Yes.

Reply:
as for the fields api, maybe to be future safe. (I am unsure what I would recommend)

Reply:
> > Does an async function always return a promise wrapping whatever you return?
> 
> Yes.

yea, makes sense, just haven't considered it before.
I'll do the field one async for now as well, then we can discuss that before the official release.